### DISCUSSION TITLE
Reverification Issue

### DISCUSSION THREAD

**Discussion by ShinonKagura** [community/community#58996] [GitHub Education]
### Hi everyone,

Hi everyone,
Hello,
I received an email from you regarding: "Academic Reverification Required".

In this email I was sent a link:
"To reverify, simply return to https://education.github.com/benefits and click "submit your information again."

I wanted to fill out exactly this form which was shown to me there:
My email address: [Email@outlook.com](mailto:Email@outlook.com))
School: Institute for Learning Systems GmbH

But now I get the following message:
We require applicants of Institute for Learning Systems GmbH to use one of these school-issued email addresses to apply:

nilskleinert.de
Please select a different email address.
The strange thing is that I was already accepted 2x with exactly this email!

You've already submitted
2 requests
Approved Submitted Dec 23, 2022
Congrats! Your request for @ShinonKagura was approved and your benefits are now available.
Approved Submitted Dec 9, 2020
Congrats! Your request for @ShinonKagura was approved and your benefits are now available.

Now all of a sudden this email extension is no longer valid?
Please clarify.

With kind regards
Shinonkagura

**Comment by CoolGenius-123** [comment]
I am also Having the same issue I have also got my github student expired and Now they are no longer verifying my Student Pack with same email id.

**Comment by ShinonKagura** [comment]
I opened this thread more than 4 days ago.
I also opened a ticket.
However, no response. 
Github is ignoring my request. 

And I am sure that my concern is quite legitimate!

**Reply by bloody-777** [reply]
Did they ever get back to you? Just asking because I am experiencing the same issues.

**Reply by ShinonKagura** [reply]
Nope never. No answer nothing

**Reply by bloody-777** [reply]
So I won't get any answer either, good to know, at least now I can stop checking my inbox.

Thanks for letting me know!

**Comment by Logxn** [comment]
I have a similar issue and I wrote about it in #61620 and a indepth blog post [here](https://blog.loganthompson.de/github-impossible/).
A shame on how this is being handled right now.

**Comment by samus-aran** [comment]
Hi @ShinonKagura 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

We apologize for any inconvenience this may cause.

Regarding the difference in email domain I have raised this internally to investigate. 
I have also found your support ticket so we'll provide an update via the ticket. 👍 Sometimes, like this, users require explicit help. But we're here to assist ⭐

For anyone else ( @Logxn 👋 ) if you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍  

<img width="685" alt="Screenshot 2023-07-29 at 22 16 17" src="https://github.com/community/community/assets/4041275/76be8a8a-45ef-4e70-aa3e-2ec0006bae06">

☝️ If you are unable to see this selectable option please mention me so I can review.

**Reply by Logxn** [reply]
Hello, I have also opened up a ticket.
And "My selected school [...]" is not a valid option for me

**Reply by ShinonKagura** [reply]
@samus-aran 
Thank you for the Answer ... after a long time :P
I hope we can fix the issue very fast. You wrote:
 have also found your support ticket so we'll provide an update via the ticket. 👍
 Right now there is no answer to my ticket but i hope it will come soon.
 Thank you so far.

**Reply by samus-aran** [reply]
Hmm @Logxn  🤔 

Couple of questions:

- Which browser are you using?
- Which browser language are you using? (I assume German)

Can you do a hard refresh on that page and send me a screenshot of the dropdown. 👍 

@ShinonKagura Yeah, you will most likely get a response in the next couple of days (being the weekend) 🙂

**Reply by Logxn** [reply]
Weirdly enough it is there now.
It wasn't last week though.

One ticket I have open is 1 month old - still no response btw

**Reply by samus-aran** [reply]
Well, that's good it's visible to you now. 👍 🎉 

I'll reach out to the support team and source your ticket. The Community team primarily help folk via the discussions but I'll liaison with the support team on your ticket or [your main discussion](https://github.com/orgs/community/discussions/61620) when we have an update.

**Reply by Brandin** [reply]
@samus-aran 
I hope you are doing well. I am the primary contact for our School and a large number of students received alerts over the weekend that they need to reverify but when they access a page, they get an error relating to their email. I haven't been provided a resolution and we never had these issues before when doing our enrollment. I raised a ticket with Enterprise Support, can you have it looked into ASAP, and see if there is a bigger problem going on, as it seems to be based on the number of community responses I've noted? 
Ticket #2273415 

Thank you!
Brandin.

**Comment by ShinonKagura** [comment]
> @ShinonKagura Yeah, you will most likely get a response in the next couple of days (being the weekend) 🙂

Last week you said someone would contact me .. that was last week. Now its weekend again.. nobody was contacting me .. Nice work XD

**Comment by c-ryan-z** [comment]
I also encountered this Issue I hope it get fixed soon.

**Comment by ShinonKagura** [comment]
I was waiting for month now. My Subscriptions expired and i cant  submit a new request. Nobody answers my Tickets .. nothing.. Wow, Nice work, Github.

**Reply by Logxn** [reply]
Same here. Two months without an answer. Its a shame

**Comment by ShinonKagura** [comment]
Congrats! Your request for [@) was approved and your benefits will be available soon.
How loang does it take to reactivate my benefits ... i mean, i wait for days now...
